packages/shared/src/components/LazyImage.tsx

```diff
@@ -52,7 +49,7 @@ function LazyImageComponent(
   };
   if (eager) {
     imageProps = { src, className: baseImageClass };
-  } else if (asyncImageSupport) {
```

Comment:
Lol if (true) "yes" 🤣

Reply:
It was fun! :D 
I tried to go deeper with refactor and now we do not need this if anymore!